.rubocop.yml

```diff
@@ -11,6 +11,9 @@ AllCops:
 RSpec/ExampleLength:
   Enabled: false
 
+RSpec/MultipleExpectations:
+  Enabled: false
```

Comment:
Can you disable this for feature specs only? Aside from feature specs, I think it makes sense for tests to only contain one expectation.